models/comments-model.js

```diff
@@ -9,6 +9,20 @@ const selectComments = (article_id) => {
                             })
 };
 
-module.exports = { selectComments }
+const insertComment = (article_id, newComment) => {
+    const {username, body} = newComment;
+    const votes = 0;
+    const created_at = new Date();
```

Comment:
Votes and created at don't need to be added manually as they have default values in the PSQL seed file